### PR TITLE
Add code with security and performance issues

### LINKED ISSUE #1: Add a few performance and security issues

### PR DESCRIPTION
**Pull request by siddhant-deepsource**
Fixes #1 when gets merged.

### CODE REVIEW

File: sync_example.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import (
+	"sync"
+	"testing"
+)
+
+var dataLarge []byte
+
+const size = 64 * 1024 //65536
+
+func Benchmark_LargeSize_Pool_ReturnNonpointer(b *testing.B) {
```
Comment (sourya-deepsource) [inline sync_example.go:12]: This should be just camel case, not a mix.

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module github.com/deepsourcelabs/demo-go
+
+go 1.13
```
Comment (sourya-deepsource) [inline go.mod:3]:
```suggestion
go 1.13

require golang.org/x/crypto v0.0.0-20201116153603-4be66e5b6582
```

**Review by sourya-deepsource — CHANGES_REQUESTED** [review]

File: sync_example.go (hunk 1 above)
Comment (siddhant-deepsource) [reply sync_example.go]: Done.

File: sync_example.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"sync"
+	"testing"
+)
+
+var dataLarge []byte
+
+const size = 64 * 1024 //65536
```
Comment (sourya-deepsource) [inline sync_example.go:10]:
```suggestion
const size = 64 * 1024 // 65536
```

**Review by sourya-deepsource — APPROVED** [review]

**Pull request merged**